packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -231,6 +297,8 @@ export function SelectPanel({
             items={itemsToRender}
             textInputProps={extendedTextInputProps}
             inputRef={inputRef}
+            loading={isLoading}
+            loadingType={loadingType()}
```

Comment:
Aw, not ideal to have both loading and loadingType in the API but I guess we'll need to keep them for backward compatibility.